prql-compiler/snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: prqlc # you probably want to 'snapcraft register <name>'
+base: core20 # the base snap is the execution environment for this snap
+version: "0.1" # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Lower priority but before we merge — we should have this be correct version and have something like this in the `prqlc` Cargo.toml so it gets update on every release:

https://github.com/PRQL/prql/blob/b430f03e05171c898ece420674d63511d17f2d69/prql-js/Cargo.toml#L44-L48

Reply:
An alternative would be to set the version property to `git` which would expand it using the `git describe` command to something like `0.2.5-690-g311ccd9` as per the documentation. https://snapcraft.io/docs/snapcraft-yaml-reference

Reply:
Great, good idea re `git`; this will give us a good result when we're on a tag